### DISCUSSION TITLE
tacotron2学習モデルの、FastSpeech2によるfine-tuningが上手くいきません

### DISCUSSION THREAD

**Discussion by Toshiknob** [espnet/espnet#5211] [Q&A]
初めまして。自分は用意したデータセットからFastSpeech2で音声合成を行いたいと考え、tacotron2モデルを学習しました。
手順は
・データ用意
・run.shをstage6まで実行し、tacotron2モデルを学習（コマンドは以下）
`./run.sh     --stage 6     --g2p pyopenjtalk_accent_with_pause     --min_wav_duration 0.38     --fs 44100     --n_fft 2048     --n_shift 512     --dumpdir dump/44k     --win_length null     --feats_extract linear_spectrogram     --feats_normalize none --train_config ./conf/tuning/finetune_tacotron2.yaml --train_args "--init_param downloads/0afe7c220cac7d9893eea4ff1e4ca64e/exp/tts_train_tacotron2_raw_phn_jaconv_pyopenjtalk_accent_with_pause/train.loss.ave_5best.pth:::tts.dec.feat_out,tts.dec.prenet.prenet, --batch_size 8 --ngpu 1" --tag finetune_tacotrion2_raw_phn_jaconv_pyopenjtalk_accent_with_pause --nj 4`
その後、
・FastSpeech2のfine-tuningを行うべく、以下のコマンドを実行しました
`./run.sh     --stage 7  --g2p pyopenjtalk_accent_with_pause       --min_wav_duration 0.38       --fs 44100       --n_fft 2048       --n_shift 512       --dumpdir dump/44k       --win_length null       --feats_extract linear_spectrogram       --feats_normalize none --tts_exp exp/tts_finetune_tacotrion2_raw_phn_jaconv_pyopenjtalk_accent_with_pause --test_sets "tr_no_dev dev eval1"   --inference_args "--use_teacher_forcing true"`
が、
`run.pl: 22 / 32 failed, log is in exp/tts_finetune_tacotrion2_raw_phn_jaconv_pyopenjtalk_accent_with_pause/decode_use_teacher_forcingtrue_train.loss.ave/tr_no_dev/log/tts_inference.*.log`
のエラーが出てしまい、実行されません。
--nj 4を足したりしましたが、同様です。tts_inferenceのlogを見ても特にエラーが書かれておらず、原因が分かりません。

・何か手順が間違っているのか
・エラーの原因や、解決法
について何かご存知でしたらご教示いただきたいです。

初歩的な質問で恐縮ですが、何卒よろしくお願いいたします。

**Comment by kan-bayashi** [comment]
```
run.pl: 22 / 32 failed, log is in exp/tts_finetune_tacotrion2_raw_phn_jaconv_pyopenjtalk_accent_with_pause/decode_use_teacher_forcingtrue_train.loss.ave/tr_no_dev/log/tts_inference.*.log
```
これは32個のジョブのうち22個が失敗していることを表しています
```
grep -e failed -e error exp/tts_finetune_tacotrion2_raw_phn_jaconv_pyopenjtalk_accent_with_pause/decode_use_teacher_forcingtrue_train.loss.ave/tr_no_dev/log/tts_inference.*.log
```
のコマンドでなにか引っかかりませんかね？

**Reply by Toshiknob** [reply]
ご回答ありがとうございます、
いただいたコマンドを入力しましたが、何も出力されませんでした。

ただ、どこで失敗しているのかよく確認するためにもう一度tacotron2のstage6で学習を実行したところ、最後に`tts_inference.py: error: No such file: conf/decode.yaml`と出ていたのを見逃していました。
stage7でなく、stage6時点で問題がある様なので、tacotron2の学習から見直します、お騒がせしてしまい申し訳ありませんでした。
